### PR TITLE
feat: update button styles

### PR DESCRIPTION
**Pull request by gregogun**
This pr corrects button sizing to match the components in Figma.

### CODE REVIEW

File: sigle/src/ui/Button.ts [hunk 1]
```diff
@@ -13,12 +13,17 @@ export const Button = styled('button', {
...
+    fontSize: '$2',
+  },
+
   variants: {
     size: {
       md: {
         px: '$3',
         py: '$1',
         br: '$1',
+        maxHeight: 28,
```
Comment (pradel) [inline sigle/src/ui/Button.ts:26]: if we set this max height prop here we can't have multi-line buttons anymore, so doesn't seems like the right way to fix the problem no ?

File: sigle/src/ui/Button.ts [hunk 2]
```diff
@@ -13,12 +13,17 @@ export const Button = styled('button', {
     pointerEvents: 'none',
   },
 
+  '@xl': {
```
Comment (pradel) [inline sigle/src/ui/Button.ts:16]: xl or md?

File: sigle/src/ui/Button.ts (hunk 1 above)
Comment (gregogun) [reply sigle/src/ui/Button.ts:26]: hmm yeah that's true - will see with Quentin if it's better to update the design then as we inspected on devtools and all properties are the same so we have no idea where the extra pixels are coming from 😅

File: sigle/src/ui/Button.ts (hunk 2 above)
Comment (gregogun) [reply sigle/src/ui/Button.ts:16]: following the breakpoints on typography which are changing at `xl` but maybe it should be `md`? as most other component styles are updating at this breakpoint. Wdyt?

Comment (pradel) [reply sigle/src/ui/Button.ts:16]: I think md make more sense, xl is really for big screens

File: sigle/src/ui/Button.ts (hunk 1 above)
Comment (pradel) [reply sigle/src/ui/Button.ts:26]: that's a really strange one, not the first time I see smth like this 😓

File: sigle/src/ui/Button.ts (hunk 2 above)
Comment (gregogun) [reply sigle/src/ui/Button.ts:16]: yeah makes sense - should I also update the typography breakpoints here, or do it in a separate pr?

Comment (pradel) [reply sigle/src/ui/Button.ts:16]: here sounds good 👍

**Comment by gregogun** [comment]
Work for this was carried over and completed in #753

**Comment by pradel** [comment]
Can we close?

**Comment by gregogun** [comment]
> Can we close?

yep all good to close 👍

**Pull request closed without merging**